tsconfig.json

```diff
@@ -12,5 +12,18 @@
     "skipLibCheck": true,
     "resolveJsonModule": true
   },
-  "include": ["./src"]
+  "include": [
+    "./src/**/*",
+    "./plugins/**/*",
+    "./gatsby-node.ts",
+    "./gatsby-config.ts",
+    "./overrides.d.ts"
+  ],
+  "exclude": [
+    "node_modules",
+    "./src/content",
+    "./src/intl",
+    "./src/data",
+    "./src/assets"
+  ]
```

Comment:
Excluded many big folders we know are not going to have `.ts` files to optimize the scanning.